src/Ng/Jobs/MonitoringProcessorJob.cs

```diff
@@ -227,37 +219,6 @@ private async Task ProcessPackagesAsync(CancellationToken token)
             await _statusService.UpdateAsync(status, token);
         }
 
-        private async Task<IEnumerable<CatalogIndexEntry>> FetchCatalogIndexEntriesFromRegistrationAsync(
```

Comment:
Fetching the catalog entry from registration is a decent mitigation to when we don't have the latest catalog entry but it could lead to correctness issues.

For example, if the package was processed by flat-container but skipped by registration, fetching the entries from registration could lead to us getting the wrong catalog entry and then skipping running validations.

Given we now no longer require catalog entries to run validations, I felt that removing this mitigation was better than leaving it in.